### ISSUE TITLE
Upgrading tippyjs

### ISSUE THREAD

**Issue by omgoshjosh** [atomiks/tippyjs-react#214]
Hello, I have upgraded tippyjs and don't really understand (possibly other devs at my company, i'm still investigating) why the following issue with the out-of-the-box popper/tippy arrow has changed.

![image](https://user-images.githubusercontent.com/1960929/83464774-af00d000-a426-11ea-8ffe-4c7187193a82.png)

like i said, i am still investigating any changes on my end that have caused this, but if there are any insights as to how or why this might be happening or what i need to do to fix it, please advise.

thank you very much. tippy and its react version are very nice.

**Comment by atomiks** [comment]
Can't determine without any repro. 

Out of the box should work fine - ensure you aren't applying a custom theme or read the migration guide about the new theming attributes/classnames.

**Comment by omgoshjosh** [comment]
Thanks for closing this. We resolved the issue and it was on our end.

For posterity, our problem was that, in addition to tippy @v5 and for backwards compatibility, we had tippy @v2 still around and it was providing global css from a single JS import.

What that meant was that when we removed the old version the issue above was seen.

To resolve it, we had to reintroduce styles by making some fixes in our webpack build to properly provide the right styles.

Beyond that it was a simple case of just updating our use of tippy to the latest APIs instead of 2+ year old APIs from v2.

@atomiks, great work you have here with tippy and the react bindings. Thanks a lot.
